### ISSUE TITLE
Perhaps more descriptive names would be clearer?

### ISSUE THREAD

**Issue by Joshua-Ronan-Phillips** [fcscripters/react-final-countdown#5]
Great module! On the whole naming was ok but I'm a big fan of always being descriptive, update: https://github.com/fcscripters/react-final-countdown/blob/master/react-countdown.jsx#L14 
could be named updateTime, as while the code on that line is self explanatory, 
https://github.com/fcscripters/react-final-countdown/blob/master/react-countdown.jsx#L114
it becomes less clear. Here:
https://github.com/fcscripters/react-final-countdown/blob/master/react-countdown.jsx#L89
the same problem occurs, small issues now but if you get into this habit as projects get bigger they will be really hard to follow.

**Comment by thegsi** [comment]
Changed function name to updateTime

**Comment by Joshua-Ronan-Phillips** [comment]
well played
